test/unit_test/snpe/test_adb_run.py

```diff
@@ -36,22 +36,34 @@ def test_run_adb_command(mock_run_subprocess, mock_which, android_target):
 
 
 def test_run_snpe_command():
-    os.environ["SNPE_ROOT"] = "C:\\snpe"
+    if platform.system() == "Windows":
+        os.environ["SNPE_ROOT"] = "C:\\snpe"
+        target_arch = "x86_64-windows-msvc"
+    else:
+        os.environ["SNPE_ROOT"] = "/snpe"
+        target_arch = "x86_64-linux-clang"
+
     with patch.object(Path, "exists") as mock_exists, patch.object(Path, "glob") as mock_glob, patch(
         "shutil.which"
     ) as mock_witch, patch("subprocess.run") as mock_run_subprocess:
         mock_exists.return_value = True
-        mock_glob.return_value = [Path("lib") / "lib/x86_64-windows-vc19"]
+        mock_glob.return_value = [Path("lib") / target_arch]
         mock_witch.side_effect = lambda x, path: x
         mock_run_subprocess.return_value = CompletedProcess(None, returncode=0, stdout=b"stdout", stderr=b"stderr")
-        stdout, stderr = run_snpe_command("snpe-net-run --container xxxx")
+        runner = SDKRunner(platform="SNPE")
+        stdout, _ = runner.run(cmd="snpe-net-run --container xxxx")
         if platform.system() == "Linux":
             env = {
-                "LD_LIBRARY_PATH": "C:\\snpe/lib/x86_64-linux-clang",
-                "PATH": "C:\\snpe/bin/x86_64-linux-clang:/usr/bin",
+                "LD_LIBRARY_PATH": "/snpe/lib/x86_64-linux-clang",
+                "PATH": "/snpe/bin/x86_64-linux-clang:/usr/bin",
+                "PYTHONPATH": "/snpe/lib/python",
+                "SDK_ROOT": "/snpe",
             }
         else:
-            env = {"PATH": "C:\\snpe\\bin\\x86_64-windows-vc19;C:\\snpe\\lib\\x86_64-windows-vc19"}
+            env = {
```

Comment:
do we need test case for path like: C:/snpe/bin/xxx etc? in Windows, the slash also works.

Reply:
Updated. Waiting CI UT for double check.

Reply:
In windows ut, the path is printed with `\\` to concatenate the path str. 
Reverted the changes.